### PR TITLE
Feature/iden 378

### PR DESCRIPTION
**Pull request by vmidyllic**
Support anoncrypt with P256 key for wrapping ECDH-ES+A256KW alg.

### CODE REVIEW

File: mock/proving.go [hunk 1]
```diff
@@ -80,3 +83,31 @@ func PrepareAuthInputs(hash []byte, id *core.ID, circuitID circuits.CircuitID) (
...
+}
+
+// ResolveEncPrivateKey returns mocked private key
+func ResolveEncPrivateKey(_ string) (interface{}, error) {
+	seed := new(big.Int)
+	seed.SetString(MockRecepientKeyID, 16)
+
+	recipientPrivKey := new(ecdsa.PrivateKey)
+	curve := elliptic.P256()
+	recipientPrivKey.PublicKey.Curve = curve
```
Comment (ilya-korotya) [inline mock/proving.go:109]: `recipientPrivKey.PublicKey.Curve = elliptic.P256()`

File: packager_test.go [hunk 2]
```diff
@@ -123,16 +123,54 @@ func TestPackagerZKPPacker(t *testing.T) {
...
+func TestPackagerAnonryptPacker(t *testing.T) {
+	pm := iden3comm.NewPackageManager()
+	pm.RegisterPackers(packers.NewAnoncryptPacker(mock.ResolveEncPrivateKey), &packers.PlainMessagePacker{})
+	// nolint :
+
+	identifier := "119tqceWdRd2F6WnAyVuFQRFjK3WUXq2LorSPyG9LJ"
+
+	_, err := core.IDFromString(identifier)
+	assert.NoError(t, err)
+	var msg protocol.CredentialFetchRequestMessage
```
Comment (ilya-korotya) [inline packager_test.go:144]: Colud you build the message in one place?
like:
```go
msg := {
  "From":"...",
  "To":"..."
  etc... 
}
```

File: packers/anoncrypt_test.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	var msgBytes = []byte(`{"id":"123","type":"https://iden3-communication.io/authorization/1.0/response","from":"119tqceWdRd2F6WnAyVuFQRFjK3WUXq2LorSPyG9LJ","body":{"scope":[{"type":"zeroknowledge","circuit_id":"auth","pub_signals":["1","18311560525383319719311394957064820091354976310599818797157189568621466950811","323416925264666217617288569742564703632850816035761084002720090377353297920"],"proof_data":{"pi_a":["11130843150540789299458990586020000719280246153797882843214290541980522375072","1300841912943781723022032355836893831132920783788455531838254465784605762713","1"],"pi_b":[["20615768536988438336537777909042352056392862251785722796637590212160561351656","10371144806107778890538857700855108667622042215096971747203105997454625814080"],["19598541350804478549141207835028671111063915635580679694907635914279928677812","15264553045517065669171584943964322117397645147006909167427809837929458012913"],["1","0"]],"pi_c":["16443309279825508893086251290003936935077348754097470818523558082502364822049","2984180227766048100510120407150752052334571876681304999595544138155611963273","1"],"protocol":""}}]}}`)
+
+	privKey, err := mock.ResolveEncPrivateKey("myencryptionkey")
```
Comment (ilya-korotya) [inline packers/anoncrypt_test.go:17]: move `myencryptionkey` and `myencryptionkey` to constant

Comment (vmidyllic) [reply packers/anoncrypt_test.go:17]: used another constant instead

File: packager_test.go (hunk 2 above)
Comment (vmidyllic) [reply packager_test.go:144]: refactored

File: mock/proving.go (hunk 1 above)
Comment (vmidyllic) [reply mock/proving.go:109]: refactored

**Review by ilya-korotya — APPROVED** [review]

**Pull request merged**